src/screens/Processors/DummyProcessorModal.res

```diff
@@ -0,0 +1,129 @@
+          {heading->React.string}
+        </h2>
+      </AddDataAttributes>
+      <div className="flex w-full justify-between">
+        <RenderIf condition={showSearch}>
+          <AddDataAttributes attributes=[("data-testid", "search-processor")]>
+            <input
+              ref={searchRef->ReactDOM.Ref.domRef}
+              type_="text"
+              value=searchedConnector
+              onChange=handleSearch
+              placeholder="Search a processor"
+              className={`rounded-md px-4 py-2 focus:outline-none w-1/3 border`}
+              id="search-processor"
+            />
+          </AddDataAttributes>
+        </RenderIf>
+        <RenderIf condition={showDummyConnectorButton}>
+          <ACLButton
+            access={userPermissionJson.connectorsManage}
+            text="+ Connect a Dummy Processor"
+            buttonType={Transparent}
+            buttonSize={Small}
+            textStyle="text-jp-gray-900"
+            // onClick={_ => {
+            //   setProcessorModal(_ => true)
+            // }}
+          />
+        </RenderIf>
+        <CantFindProcessor showRequestConnectorBtn setShowModal />
+      </div>
+      <RenderIf condition={connectorList->Array.length > 0}>
+        <div
+          className="grid gap-x-5 gap-y-6 2xl:grid-cols-4 lg:grid-cols-3 md:grid-cols-2 grid-cols-1 mb-5">
+          {connectorList
+          ->Array.mapWithIndex((connector: ConnectorTypes.connectorTypes, i) => {
+            let connectorName = connector->getConnectorNameString
+            let connectorInfo = connector->getConnectorInfo
+            let size = "w-14 h-14 rounded-sm"
+
+            <ACLDiv
+              permission={userPermissionJson.connectorsManage}
+              onClick={_ => handleClick(connectorName)}
+              key={i->string_of_int}
+              className="border p-6 gap-4 bg-white rounded flex flex-col justify-between"
+              dataAttrStr=connectorName>
+              <div className="flex flex-col gap-3 items-start">
+                <GatewayIcon gateway={connectorName->String.toUpperCase} className=size />
+                <p className={`${p1MediumTextStyle} break-all`}>
+                  {connectorName
+                  ->getDisplayNameForConnector(~connectorType=ConnectorTypes.Processor)
+                  ->React.string}
+                </p>
+              </div>
+              <p className="overflow-hidden text-gray-400 flex-1 line-clamp-3">
+                {connectorInfo.description->React.string}
+              </p>
+              <ACLButton
+                access={userPermissionJson.connectorsManage}
+                text="+ Connect"
+                buttonType={Transparent}
+                buttonSize={Small}
+                textStyle="text-jp-gray-900"
+              />
+            </ACLDiv>
+          })
+          ->React.array}
+        </div>
+      </RenderIf>
+      <RequestConnector connectorList setShowModal />
+    </>
+  }
+
+  <Modal
```

Comment:
Add a render if over the modal to render only when it's true.

It's our component fix that we need to make but for now - 

<RenderIf condition={processorModal}>
...
</Renderif>

Reply:
Add this